jobs/e2e-test.groovy

```diff
@@ -484,7 +484,7 @@ def runLambda(){
    // In case the main provider is down, shift a build-job to 'e2e-test-cypress-backup'
    build(job: 'e2e-test-cypress',
           parameters: [
-             string(name: 'SLACK_CHANNEL', value: "#cypress-testing"),
+             string(name: 'SLACK_CHANNEL', value: params.SLACK_CHANNEL),
```

Comment:
**note:** By doing this change, we are now going to send the Cypress results to the #1s-and-0s-deploys channel.